src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/SessionType.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.model
+
+enum class SessionType(
```

Comment:
named enum as SessionType instead on SessionRestriction as it might be confused with VisitRestriction.